articles/aks/operator-best-practices-multi-region.md

```diff
@@ -58,6 +59,12 @@ For information on how to set up endpoints and routing, see [Configure the geogr
 
 Traffic Manager uses DNS (layer 3) to shape traffic. [Azure Front Door Service](https://docs.microsoft.com/azure/frontdoor/front-door-overview) provides an HTTP/HTTPS (layer 7) routing option. Additional features of Azure Front Door Service include SSL termination, custom domain, web application firewall, URL Rewrite, and session affinity. Review the needs of your application traffic to understand which solution is the most suitable.
 
+### Interconnect regions with global vnet peering
+
+If the clusters need to talk to each other, connecting both virtual networks to each other can be achieved through [Virtual Network Peering][https://docs.microsoft.com/azure/virtual-network/virtual-network-peering-overview]. This technology interconnects Virtual Networks to each other providing high bandwidth across Microsoft's backbone network, even across different geographic regions.
+
+A prerequisite to peer the virtual networks where AKS clusters are running is that you have deployed your AKS cluster in your own Virtual Network, either using kubenet or the Azure CNI. Additionally, make sure to use the standard Load Balancer in your AKS cluster, so that kubernetes services are reachable across the Virtual Network peering.
```

Comment:
@erjosito Thanks for suggesting this as this is good information to list (I learned something too!)

In the 1st paragraph, can you change the link to use the \[title\]\(link\) format?

One suggestion for the 2nd paragraph is to remove some of the unnecessary information and be more precise.  e.g. "A prerequisite to using Virtual Network Peering across two AKS clusters in different regions is to [use the standard Load Balancer](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-peering-overview#requirements-and-constraints) in both AKS clusters."  (i.e. "In your own Virtual Network, either using kubenet or the Azure CNI" is redundant)